README.md

```diff
@@ -58,9 +75,9 @@ class Application extends React.Component {
 - `edgeHitWidth` (Number) - Edge distance on content view to open side menu, defaults to 60
 - `toleranceX` (Number) - X axis tolerance
 - `toleranceY` (Number) - Y axis tolerance
-- `disableGestures` (Bool) - Disable whether the menu can be opened with gestures or not
-- `onStartShouldSetResponderCapture` (Function) - Function that accepts event as an argument and specify if side-menu should react on the touch or not. Check https://facebook.github.io/react-native/docs/gesture-responder-system.html for more details
-- `onChange` (Function) - Callback on menu open/close. Is passed `isOpen` as an argument
+- `onSwipe` (Function) - Function that handles gestures, receives boolean argument indicating whether menu should be opened or not based on the drag. When not defined, gestures are disabled
+- `onStartShouldSetResponderCapture` (Function) - Function that accepts event as an argument and specify if side-menu should react on the touch or not. Check https://facebook.github.io/react-native/docs/gesture-responder-system.html for more details. Typically you would like to set `isOpen` value to the passed argument (see examples/Basic.js)
+- `onContentPress` (Function) - Function that handles content press when menu is opened. Typically you would set isOpen to false inside it (see examples/Basic.js). When not present, content is accessible when menu is opened and there's no `touchToClose` behavior
```

Comment:
Can you explain why you think that `onChange` is not needed in this case? Sometimes people needs to know the exact moment when animation has been ended. Based on your feedback I think I can conclude that we can invoke some callback every time animated value changes (see https://facebook.github.io/react-native/docs/animations.html#input-events). It can be a good alternative to onChange, wdyt?

So literally, you want to make it possible to control an offset from the parent component? Like `<SideMenu left={20} />` or something like this? If that's a case, we can approach it a bit diffirently: by dividing this component for "core" one - which will be a 100% props-driven without any local state. The only prop it'll take is a `left` prop. And make a wrapper for dragging and make a current behavior optional (like a top-level API). What do you think, would it cover both our cases? If changing a `left` property directly would be a case - you will be able to use a `core` component, if you want a more high-level api - use a different one. And both of them may be shipped inside a side-menu like `react-native-side-menu` and `react-native-side-menu/core` (or smth like that).